src/structures/Message.js

```diff
@@ -374,22 +374,35 @@ class Message extends Base {
     async acceptGroupV4Invite() {
         return await this.client.acceptGroupV4Invite(this.inviteV4);
     }
+    
+    /**
+     * Forward options:
+     * 
+     * @typedef {Object} MessageForwardOptions
+     * @property {boolean} [multicast=false]
+     * @property {boolean} [withCaption=false] Forwards this message with the caption text of the original message if provided.
+     */
 
     /**
      * Forwards this message to another chat (that you chatted before, otherwise it will fail)
      *
      * @param {string|Chat} chat Chat model or chat ID to which the message will be forwarded
+     * @param {MessageForwardOptions} [options] Options used when forwarding the message
      * @returns {Promise}
      */
-    async forward(chat) {
+    async forward(chat, options = {}) {
         const chatId = typeof chat === 'string' ? chat : chat.id._serialized;
+        let internalOptions = {
+            multicast: options.multicast || false,
+            withCaption: options.withCaption || false
```

Comment:
I haven't tried this out yet so doing a very quick drive-by - should we default this to true? If this is the behavior from WhatsApp Web when forwarding then I would expect most people want that.

Reply:
Yes im agree with you it would be better, ill change it